terraform/modules/hub/matomo.tf

```diff
@@ -0,0 +1,536 @@
+locals {
+  volume_name = "shared_www"
+
+  log_format = <<-LOG_FORMAT
+  log_format  matomo  '{"ip": "$remote_addr",'
+                       '"http_x_forwarded_for": "$http_x_forwarded_for",'
+                       '"host": "$host",'
+                       '"path": "$request_uri",'
+                       '"status": "$status",'
+                       '"referrer": "$http_referer",'
+                       '"user_agent": "$http_user_agent",'
+                       '"length": $bytes_sent,'
+                       '"generation_time_milli": $request_time,'
+                       '"userid": "$remote_user",'
+                       '"request": "$request",'
+                       '"msec": "$msec",'
+                       '"method": "$request_method",'
+                       '"content_type": "$content_type",'
+                       '"date": "$time_iso8601"}';
+
+  access_log /dev/stdout matomo;
+LOG_FORMAT
+
+
+  nginx_log_format_base64 = base64encode(local.log_format)
+}
+
+resource "aws_kms_key" "matomo" {
+  description = "used to encrypt secrets in parameter store for matomo"
+}
+
+# resource "aws_kms_alias" "matomo" {
```

Comment:
why is this commented out?

Reply:
doesn't appear used, but exists in the old world